### PR TITLE
Fixed #28260 -- Allowed customizing the test tablespace initial and autoextend size on Oracle.

### PR DESCRIPTION
**Pull request by felixxm**
https://code.djangoproject.com/ticket/28260

### CODE REVIEW

**Comment by felixxm** [comment]
buildbot, test on oracle.

**Comment by felixxm** [comment]
buildbot, test on oracle.

File: docs/releases/2.0.txt [hunk 1]
```diff
@@ -361,6 +365,11 @@ Miscellaneous
   ``PASSWORD_RESET_TIMEOUT_DAYS = 3``, password reset tokens are now valid for
   72 hours rather than 96 hours.
 
+* The default size of the Oracle test tablespace has increased from 20M to 50M.
```
Comment (timgraham) [inline docs/releases/2.0.txt]: Single bullet point seems okay: "The default size of the Oracle test tablespace is increased from 20M to 50M and the default autoextend size is increased ..."

File: docs/ref/settings.txt [hunk 2]
```diff
@@ -911,6 +911,74 @@ This is an Oracle-specific setting.
...
+.. setting:: DATAFILE_SIZE
+
+``DATAFILE_SIZE``
+^^^^^^^^^^^^^^^^^
+
+.. versionadded:: 2.0
+
+Default: ``'50M'``
+
+.. versionchanged:: 2.0
```
Comment (timgraham) [inline docs/ref/settings.txt]: I think the changed annotations/notes can be omitted. Most users are unlikely to care, I'd guess.

File: docs/releases/2.0.txt [hunk 3]
```diff
@@ -234,6 +234,10 @@ Tests
 
 * Added threading support to :class:`~django.test.LiveServerTestCase`.
 
+* Added settings that allow customization of test tablespace parameters for
```
Comment (timgraham) [inline docs/releases/2.0.txt]: "allow customizing the test tablespace parameters"

File: docs/releases/2.0.txt [hunk 4]
```diff
@@ -234,6 +234,10 @@ Tests
...
+  Oracle: :setting:`DATAFILE_SIZE`, :setting:`DATAFILE_TMP_SIZE`,
+  :setting:`DATAFILE_EXTSIZE`, :setting:`DATAFILE_TMP_EXTSIZE`.
```
Comment (timgraham) [inline docs/releases/2.0.txt]: , and

File: docs/ref/settings.txt [hunk 5]
```diff
@@ -911,6 +911,74 @@ This is an Oracle-specific setting.
...
+
+Default: ``'25M'``
+
+.. versionchanged:: 2.0
+
+    The previous value was 10M and was not user customizable.
+
+This is an Oracle-specific setting.
+
+The size that the DATAFILE is extended when more space is required.
```
Comment (timgraham) [inline docs/ref/settings.txt]: The amount by which the DATAFILE is extended when more space is required.
(use similar wording for next note also)

**Comment by felixxm** [comment]
@timgraham Thanks for review. I updated docs.

**Pull request merged**